access_point/server/server.py

```diff
@@ -13,19 +13,19 @@ def install():
 
 @app.route('/config/<stt>/<tts>/<hotword>/<wake>')
 def config(stt, tts, hotword, wake):
-    os.system('sudo bash /home/pi/SUSI.AI/susi_linux/access_point/server/config.sh {} {} {} {}'.format(stt,tts,hotword,wake))  #nosec #pylint-disable type: ignore
+    subprocess.call(['sudo', 'bash' , '/home/pi/SUSI.AI/susi_linux/access_point/server/config.sh {} {} {} {}'.format(stt,tts,hotword,wake)])  #nosec #pylint-disable type: ignore
```

Comment:
Replace with

```py
subprocess.call(['sudo', 'bash' , '/home/pi/SUSI.AI/susi_linux/access_point/server/config.sh', stt, tts, hotword, wake])
```

Or

```py
subprocess.call(['sudo', 'bash' , 'server/config.sh', stt, tts, hotword, wake])
```

to not stick to location of _susi_linux_.

Reply:
ok @hongquan making the changes wherever this exists